pypit/artrace.py

```diff
@@ -1485,6 +1485,11 @@ def pad_dict(indict):
             badlines += 1
             trcdict = pad_dict(trcdict)
             continue
+        if j == 74:  # KLUDGE!!!!
```

Comment:
commented out.  I was just testing but I worry 
it might need to rear its ugly head again someday..